lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -425,13 +425,43 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
 };
 
 WebMapServiceCatalogItem.prototype._load = function() {
+    var that = this;
+    var promises = [];
+    
     if (!defined(this._rawMetadata)) {
-        var that = this;
-        return loadXML(proxyCatalogItemUrl(this, this.metadataUrl)).then(function(xml) {
+        promises.push(loadXML(proxyCatalogItemUrl(this, this.metadataUrl)).then(function(xml) {
             var metadata = capabilitiesXmlToJson(xml);
             that.updateFromCapabilities(metadata, false);
-        });
+        }));
+    }
+        
+    // Query WMS for wfs or wcs URL if no dataUrl is present
+    if (!defined(this.dataUrl)) {
+        var describeLayersURL = cleanUrl(this.url) + '?service=WMS&version=1.1.1&request=DescribeLayer&Layers=' + encodeURIComponent(this.layers);
```

Comment:
A few little questions on this line - Table 1 of the pdf document in #462 refers to version=1.3.0 and sld_version=1.1.1, saying both fields are mandatory. Also, can I suggest consistent capitalisation, ie. use "layers=" instead of "Layers="? (The spec says "LAYERS" but we use lower case everywhere else.)